prime-router/src/main/kotlin/Organization.kt

```diff
@@ -55,29 +55,7 @@ open class Organization(
      * Validate the object and return null or an error message
      */
     fun consistencyErrorMessage(): String? {
-        return when (jurisdiction) {
-            Jurisdiction.FEDERAL -> {
-                if (stateCode != null || countyName != null) {
-                    "stateCode or countyName not allowed for FEDERAL organizations"
-                } else {
-                    null
-                }
-            }
-            Jurisdiction.STATE -> {
-                if (stateCode == null || countyName != null) {
-                    "stateCode required for STATE organizations"
-                } else {
-                    null
-                }
-            }
-            Jurisdiction.COUNTY -> {
-                if (stateCode == null || countyName == null) {
-                    "stateCode and countyName required for COUNTY organizations"
-                } else {
-                    null
-                }
-            }
-        }
+        return null
```

Comment:
Can we just remove this function now?  Always returning `null` doesn't seem to provide any value?

Reply:
I am reluctant to remove the consistencyErrorMessage() which is implemented by Organization, Sender, Receiver, it seems that it is expected to do some settings creation time checking to prevent a bad setting Object got created and causing issue at the message processing time.

the goal of this PR is just removing the validation that the settings yaml schemas could do (and suppose to do) from  the function, for things that beyond yaml schemas validation, they should stay there...

Reply:
I'm fine if you don't want to remove it, but at least we need to add a tech debt ticket to follow up and clean this up.  As it stands the code will now be confusing since we have functions that always return `null`

Reply:
cleanup the unused func across, can come up with similar thing if there is a need in future.